var/spack/repos/builtin/packages/gaudi/package.py

```diff
@@ -11,10 +11,16 @@ class Gaudi(CMakePackage):
 
     homepage = "http://gaudi.web.cern.ch/gaudi/"
     git      = "https://gitlab.cern.ch/gaudi/Gaudi.git"
+    url      = "https://gitlab.cern.ch/gaudi/Gaudi/-/archive/v33r1/Gaudi-v33r1.tar.gz"
 
-    version('develop', branch='master')
-    version('30.5',    commit='2c70e73ee5b543b26197b90dd59ea4e4d359d230')
-    version('31.0',    commit='aeb156f0c40571b5753a9e1dab31e331491b2f3e')
```

Comment:
We don't normally remove old releases from packages in case someone still depends on them.